app/models/mediaflux/http/namespace_create_request.rb

```diff
@@ -27,7 +27,12 @@ def self.service
         def build_http_request_body(name:)
           super do |xml|
             xml.args do
-              xml.namespace namespace if namespace.present?
+              if namespace.present?
+                xml.namespace do
+                  xml.parent.set_attribute("all", true)
```

Comment:
what does the `all=true' do?

Reply:
It makes the parent namespaces if they do not exists.  It essentially allows the code to run on an empty docker instance as long as the schema has been created.

Reply:
Great, I didn't know we could do this. It makes the code so much flexible!